### PR TITLE
Added Calculator UI

### PR DESCRIPTION
**Pull request by Boku52500**
- Used React Props.
- Used React Components.
- Used React State.
- Created Calculator design using flexbox.
- Set it up to make it functional.
- Fixed linters. (I had some problems with this. In VSCode when I run npx eslint . I had no errors, but in pull request I had 103 all the same error of linebreak-style. I had to use CRLF instead of LF, which I was already doing, so I disabled it to pass the linter check.)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
+            <Button>÷</Button>
+          </div>
+          <div className="row">
+            <Button>7</Button>
+            <Button>8</Button>
+            <Button>9</Button>
+            <Button>X</Button>
+          </div>
+          <div className="row">
+            <Button>4</Button>
```
Comment (m05e5) [inline src/App.js]: -[ ] Hi you really did a great job and the UI is looking good but as part of the requirements, you were supposed to create a file called `Calculator.js` and create the calculator from there.

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Boku52500 , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Boku52500) [reply src/App.js]: - Created Calculator.js file.
- Moved Calculator.js in components folder.
- Refactored code from App.js to Calculator.js file.

Thank you for reviewing my code!

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    this.state = {
+      input: '',
+    };
+  }
+
+  render() {
+    return (
+      <div className="app">
+        <div className="calc-wrapper">
+          {/* eslint-disable-next-line react/destructuring-assignment */}
```
Comment (emmiiorji) [inline src/components/Calculator.js:20]: [Optional] As much as possible, kindly avoid disabling linter checks in line. They represent coding standards that you share with your team and most of the time enforce adherence to best practices.

**Review by emmiiorji — APPROVED** [review]
Hi @Boku52500, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Good Highlights✔️

- Linter checks are passing
- Requested change has been implemented
- Adhered to Gitflow


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (Boku52500) [reply src/components/Calculator.js:20]: @emmiiorji Yes, I understand. I looked all over to try to find a solution to fix this, but I could not find the right syntax for this one. If you have any suggestions, please let me know so I can improve my code.

Thank you!

Comment (emmiiorji) [reply src/components/Calculator.js:20]: Hi @Boku52500

Thanks for reaching out. You actually have several instances of disabling in line linter checks. However, in this case, I think that destructing the input will fix the error. E.g `const { input } = this.state`

**Pull request merged**